platformio.ini

```diff
@@ -12,7 +12,7 @@
 platform = espressif8266
 framework = arduino
 board_f_cpu = 160000000L
-build_flags = !python scripts/firmware_version.py -Wall -DARDUINOJSON_ENABLE_PROGMEM=1
+build_flags = !python scripts/firmware_version.py -Wall
```

Comment:
Have you tested this? It crashed frequently in my setup without. If it works, it works with an extra implicit string conversion which is more expensive (and could have been the cause of the crash due to heap fragmentation). I highly recommend to set this flag!
See https://arduinojson.org/api/config/enable_progmem/

Reply:
Yes, for sure i tested this. 
The problem was simply, that in `Settings.h` ArduinoJson was loaded before WString: db3e9d53b2d475be8f2b8507402b8206673197f6
In this case, ArduinoJson will not enable PROGMEM support.

Please show me where `ARDUINOJSON_ENABLE_PROGMEM=1`will change the behavior. As far I understand it, this flag is set automatically, if WString.h was included before.